### PR TITLE
Update production-deployment.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to #check license state

fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10806

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/production-deployment.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ In this deployment scenario, you'll be guided through the steps on:
 
 ## Check license state
 
-Checking for the license state and whether it got properly provisioned, can be done through the admin center or through the **Microsoft Azure portal**.
+Checking for the license state and whether it was properly provisioned, can be done through the admin center or through the **Microsoft Azure portal**.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/production-deployment.md]:
```suggestion
Checking for the license state and whether it was properly provisioned can be done through the admin center or through the **Microsoft Azure portal**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**